src/utils/redirectUtils.ts

```diff
@@ -17,3 +22,28 @@ export const redirectToErrorPage = () => {
     console.warn("Prevented redirect to /error page");
   }
 };
+
+export const handleRedirectIfNotValid = (
+  userResult: UserResult,
+  envs?: Envs,
+) => {
+  if (userResult.statusCode === 401) {
```

Comment:
maybe lets use constatnts instead of pure numbers

Reply:
fixed